colorAdder.js

```diff
@@ -0,0 +1,29 @@
+const colors = ['red', 'blue', 'green', 'yellow', 'black']; 
+const objs = [{id: 1, name: 'Alex'}, 
+{id: 2, name: 'Oleg'}, 
+{id: 3, name: 'Vasiliy'}, 
+{id: 4, name: 'Sergey'}, 
+{id: 5, name: 'Vladislav'}, 
+{id: 6, name: 'Grigoriy'}, 
+{id: 7, name: 'Anastasia'}, 
+{id: 8, name: 'Egor'}, 
+{id: 9, name: 'Svetlana'}, 
+{id: 10, name: 'Lana'}, 
+{id: 11, name: 'Cat'}, 
+{id: 12, name: 'Vyacheslav'}, 
+{id: 13, name: 'Ibragim'}, 
+{id: 14, name: 'Olga'}, 
+{id: 15, name: 'Mikhail'},
+];
+
+const getRandomColor = max => { 
+  let x = Math.floor(Math.random() * max);
+  return colors[x];
+};
+
+const newObjs = objs.map(item => {
+  const container = {...item, color: getRandomColor(colors.length) };
+  return container;
```

Comment:
you can create it without return and without creating const variable. Map return new object on each iteration.

Reply:
and after this you wont need x variable.
p.s  "x" is bad name for variable. I cant understand that this variable do